waku/waku_rln_relay/group_manager/on_chain/group_manager.nim

```diff
@@ -291,13 +291,17 @@ proc getRawEvents(
   let ethRpc = g.ethRpc.get()
   let wakuRlnContract = g.wakuRlnContract.get()
 
-  var events: JsonNode
-  g.retryWrapper(events, "Failed to get the events"):
+  var eventStrs: seq[JsonString]
+  g.retryWrapper(eventStrs, "Failed to get the events"):
     await wakuRlnContract.getJsonLogs(
       MemberRegistered,
-      fromBlock = some(fromBlock.blockId()),
-      toBlock = some(toBlock.blockId()),
+      fromBlock = Opt.some(fromBlock.blockId()),
+      toBlock = Opt.some(toBlock.blockId()),
     )
+
+  var events = newJArray()
+  for eventStr in eventStrs:
+    events.add(parseJson($eventStr))
```

Comment:
I don't know where the dollar come from. But to be on safe side:
```Nim
var events = newJArray()
  for eventStr in eventStrs:
    events.add(parseJson(eventStr.string))
```

Reply:
Probably I overlooked it as an leftover here. Thanks for it.